### PR TITLE
[0.5] docs: updates related to i18n/l10n

### LINKED ISSUE #2637: Administrator guide: Needs documentation on selecting locales
## Description

We should add some documentation with a walkthrough on how an admin can use `./securedrop-admin` to select locales for the interfaces, and we should include some information on how the _default_ locale can be changed. 

## User Stories

As a SecureDrop administrator, I want clear step-by-step instructions on how I can enable my source interface in languages of interest.

### PR DESCRIPTION
**Pull request by dachary**
## Status

Ready for review

## Description of Changes

* instructions on how to create new translations 
* document language selection during install/upgrade Fixes #2637

## Testing

* vagrant ssh development
* cd /vagrant
* make docs
* firefox http://localhost:8000/development/i18n.html#creating-new-translations

## Deployment

Nope.

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

File: docs/upgrade/0.4.x_to_0.5.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+.. code:: sh
+
+   git checkout 0.5
+   git stash pop
+
+Choose the list of supported languages
+--------------------------------------
+
+You need to run the `./securedrop-admin sdconfig` command again,
+following the same instructions as during the :doc:`fisrt installation
```
Comment (freddymartinez9) [inline docs/upgrade/0.4.x_to_0.5.rst]: typo here

Comment (dachary) [reply docs/upgrade/0.4.x_to_0.5.rst]: I don't  see it ? But I'm not a native english speaker, that may be why ;-)

Comment (dachary) [reply docs/upgrade/0.4.x_to_0.5.rst]: damn s/fisrt/first/ :-)

**Comment by redshiftzero** [comment]
Let's make the base branch for this one `release/0.5` so that we get these doc updates in the stable docs

File: docs/upgrade/0.4.x_to_0.5.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+You should then proceed to update the application server with:
+
+.. code:: sh
+
+    ./securedrop-admin install
+
+.. include:: ../includes/rerun-install-is-safe.txt
+
+Verify the source interface
```
Comment (redshiftzero) [inline docs/upgrade/0.4.x_to_0.5.rst]: `Verify the source interface displays the selected languages`?

File: docs/upgrade/0.4.x_to_0.5.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+.. code:: sh
+
+   cd ~/Persistent/securedrop
+
+Stash your local configuration, fetch the latest code, and verify the tag for the
+latest release (0.5):
+
+.. code:: sh
+
+   git stash save "site specific configs"
```
Comment (redshiftzero) [inline docs/upgrade/0.4.x_to_0.5.rst]: Admins shouldn't need to do the stash and pop at the end (that was only needed in 0.3.x before we had `site-specific` - now config is not stored in a file under version control). We can instead add a note to the top of this doc that tells admins to first follow the instructions in the 0.3->0.4 migration document if they had _not_ yet updated their admin workstation to 0.4. Else they can just proceed with fetching the new release branch and verifying the tag.

File: docs/upgrade/0.4.x_to_0.5.rst (hunk 2 above)
Comment (dachary) [reply docs/upgrade/0.4.x_to_0.5.rst]: done

File: docs/upgrade/0.4.x_to_0.5.rst (hunk 3 above)
Comment (dachary) [reply docs/upgrade/0.4.x_to_0.5.rst]: Ah, I did not realize that and was curious about this recommendation .I thought some people override files found in the repository for some reason. done

**Comment by dachary** [comment]
@redshiftzero thanks for your review. Rebased to **release/0.5** as suggested.

File: docs/development/i18n.rst [hunk 4]
```diff
@@ -193,13 +239,33 @@ requests for the `SecureDrop git repository`_.
 
 |Weblate commit Unlocked|
 
+Updating the full text index
+----------------------------
+
+The full text index can occasionally not be up to date. The symptom
+may be that the search function fails to find a work that you know
```
Comment (redshiftzero) [inline docs/development/i18n.rst]: `work` -> `word`

File: docs/install.rst [hunk 5]
```diff
@@ -25,6 +25,26 @@ on network speed and computing power.
...
+* German (de_DE)
+* Spanish (es_ES)
+* French (fr_FR)
+* Norwegian (nb_NO)
+* Dutch (nl)
+* Portuguese, Brasil (pt_BR)
+
+During the installation you will be given the opportunity to chose the
+list of supported languages to display, using the code shown in
+parenthesis. When the source interface is displayed in French (for
```
Comment (redshiftzero) [inline docs/install.rst]: `parenthesis` (singular) -> `parentheses` (plural)

File: docs/upgrade/0.4.x_to_0.5.rst [hunk 6]
```diff
@@ -0,0 +1,77 @@
+Upgrade from 0.4.x to 0.5.x
+===========================
+
+.. note::
+   First follow the instructions in the 0.3 to 0.4
+   :doc:`migration document <0.3.x_to_0.4>`
+   if you have not yet updated your admin workstation to 0.4.
```
Comment (redshiftzero) [inline docs/upgrade/0.4.x_to_0.5.rst]: `admin workstation` -> `**Admin Workstation**`

File: docs/upgrade/0.4.x_to_0.5.rst [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+Once you've verified the latest release, check it out:
+
+.. code:: sh
+
+   git checkout 0.5
+
+Choose the list of supported languages
+--------------------------------------
+
+You need to run the `./securedrop-admin sdconfig` command again,
```
Comment (redshiftzero) [inline docs/upgrade/0.4.x_to_0.5.rst]: `./securedrop-admin sdconfig` should be in double backticks to render as fixed width

File: docs/install.rst [hunk 8]
```diff
@@ -25,6 +25,26 @@ on network speed and computing power.
...
+
+* Arabic (ar)
+* German (de_DE)
+* Spanish (es_ES)
+* French (fr_FR)
+* Norwegian (nb_NO)
+* Dutch (nl)
+* Portuguese, Brasil (pt_BR)
+
+During the installation you will be given the opportunity to chose the
```
Comment (redshiftzero) [inline docs/install.rst]: `chose` -> `choose`

File: docs/development/i18n.rst (hunk 4 above)
Comment (dachary) [reply docs/development/i18n.rst]: done

File: docs/install.rst (hunk 8 above)
Comment (dachary) [reply docs/install.rst]: done

File: docs/install.rst (hunk 5 above)
Comment (dachary) [reply docs/install.rst]: done

File: docs/upgrade/0.4.x_to_0.5.rst (hunk 6 above)
Comment (dachary) [reply docs/upgrade/0.4.x_to_0.5.rst]: done

File: docs/upgrade/0.4.x_to_0.5.rst (hunk 7 above)
Comment (dachary) [reply docs/upgrade/0.4.x_to_0.5.rst]: done

**Comment by dachary** [comment]
@redshiftzero thanks for fixing my styling / spelling mistakes, much appreciated. I'll be more diligent with that in the future to not waste your time.

**Comment by redshiftzero** [comment]
no worries, that's what review is here for - thanks for writing these docs 😃

**Pull request merged**